clients/ui/frontend/src/app/pages/modelRegistry/screens/RegisteredModelsArchive/RegisteredModelsArchiveListView.tsx

```diff
@@ -77,7 +77,7 @@ const RegisteredModelsArchiveListView: React.FC<RegisteredModelsArchiveListViewP
                   icon={<FilterIcon />}
                 />
               </ToolbarFilter>
-              <ToolbarItem variant="label">
```

Comment:
As a note for anyone wondering why I removed these label variants, they were a port from the PF6 SPIKE PR we have been following which is untested. Turns out if you use the label variant on the ToolbarItem it will add the `aria-hidden='true'` property. This goes against a11y because we have a search input as a child which is focusable. Removing the label variant removes the `aria-hidden` prop.

Reply:
Ah yes, that's a great point, I'm sorry cause I faced a similar issue and I didn't report it back, dind't ring a bell yesterday when we discussed it.